README.md

```diff
@@ -82,7 +82,7 @@ Mixxx is a vibrant community of hackers, DJs and artists. To keep track of
 development and community news:
 
 - Chat with us on [Zulip][zulip].
-- Follow us on [Mastodon], [Twitter] and [Facebook].
+- Follow us on [Mastodon] and [Facebook].
```

Comment:
we replaced it with bluesky right?

Reply:
For now, we've simply removed are presence from it. I think the main social should be Mastodon, but there seem to be [an active Mixxx account on Bluesky](https://bsky.app/profile/mixxx.bsky.social) too - @ywwg are you managing this?

Reply:
yup, https://mixxx.zulipchat.com/#narrow/channel/109121-core-team/topic/bluesky (non public channel)